### PR TITLE
Remove ARID and replace estab ARID with estab UPRN

### PR DESCRIPTION
**Pull request by NickGrantX**
# Motivation and Context
ARID is dead. Long live ARID. May estab UPRN have a long and victorious reign.

# What has changed
ARID has been removed. Estab ARID has been usurped by Estab UPRN.

# How to test?
Run the ATs along with all the other PRs on the ticket.

# Links
Trello: https://trello.com/c/zm2Ssvnn

### CODE REVIEW

**Comment by NickGrantX** [comment]
[Make sample load changes to meet new sample spec (13)](https://trello.com/c/zm2Ssvnn/771-make-sample-load-changes-to-meet-new-sample-spec-13)

File: resources/sample_files/sample_input_wales_census_spec.csv [hunk 1]
```diff
@@ -1,6 +1,6 @@
...
-DDR190314000000320079,DDR190314000000114781,,HH,Household,U,RD06,,Flat 2,60 Coronation Road,,Windleybury,XX1 0XX,51.4451742,-2.5975551,E00074236,E01033361,E02003050,E06000023,E12000009,3,3,HH_LF2R1W,7,8,,0
-DDR190314000000516472,DDR190314000000516472,,HH,Household,U,RD04,,14 Blackhorse Road,,,Windleybury,XX11 0XX,51.4603426,-2.5122237,E00173063,E01015005,E02003117,E06000025,E12000009,4,2,HH_LF2R2W,10,11,,0
-DDR190314000000522356,DDR190314000000638000,,HH,Household,U,RD06,,18 Marshfield Park Cleeve Wood Road,,,Windleybury,XX11 0XX,51.4958465,-2.513198,E00075275,E01014897,E02003111,E06000025,E12000009,5,1,HH_LF2R3AW,13,14,,0
+UPRN,ESTAB_UPRN,ADDRESS_TYPE,ESTAB_TYPE,ADDRESS_LEVEL,ABP_CODE,ORGANISATION_NAME,ADDRESS_LINE1,ADDRESS_LINE2,ADDRESS_LINE3,TOWN_NAME,POSTCODE,LATITUDE,LONGITUDE,OA,LSOA,MSOA,LAD,REGION,HTC_WILLINGNESS,HTC_DIGITAL,TREATMENT_CODE,FIELDCOORDINATOR_ID,FIELDOFFICER_ID,CE_EXPECTED_CAPACITY,CE_SECURE,PRINT_BATCH
+,58493371521,HH,Household,U,RD06,,Flat 56 Francombe House,Commercial Road,,Windleybury,XX1 0XX,51.4463421,-2.5924477,E00073438,E01014540,E02003043,E06000023,E12000009,1,5,HH_LFNR2W,1,2,,0,26
```
Comment (ryangrundy7) [inline resources/sample_files/sample_input_wales_census_spec.csv]: UPRN is a mandatory value so should have a field here

**Review by LukeLoze1 — APPROVED** [review]

File: resources/sample_files/sample_input_wales_census_spec.csv (hunk 1 above)
Comment (ryangrundy7) [reply resources/sample_files/sample_input_wales_census_spec.csv]: Been addressed

**Review by ryangrundy7 — APPROVED** [review]
ATs pass for me 👍

**Review by AdamHawtin — APPROVED** [review]
All passed locally 👍

**Pull request merged**